PACKAGE = "rm_windmill_speed"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# select_bar = gen.enum([gen.const("Pass", int_t, 0, "Pass"),
#                        gen.const("Select", int_t, 1, "Select")], "type")
# gen.add("select_bar", int_t, 0, "", 1, 0, 1, edit_method=select_bar)

gen.add("is_filter_readied", bool_t, 0, "", False)
# gen.add("armor_debug_data", bool_t, 0, "", False)
# gen.add("max_angle", double_t, 0, "", 30, 0, 65)
# gen.add("min_lw_ratio", double_t, 0, "", 2.4, 0, 4)

exit(gen.generate(PACKAGE, "rm_windmill_speed", "Windmill"))